webapp/security/models.py

```diff
@@ -77,3 +149,37 @@ class Release(Base):
     version = Column(String, unique=True)
     codename = Column(String, unique=True)
     lts = Column(Boolean)
+
+
+class CVEReference(Base):
+    __tablename__ = "cve_reference"
+
+    id = Column(Integer, primary_key=True)
+    uri = Column(String)
+
+
+class Bug(Base):
+    __tablename__ = "bug"
+
+    id = Column(Integer, primary_key=True)
+    uri = Column(String)
+
+
+class Package(Base):
+    __tablename__ = "package"
+
+    id = Column(Integer, primary_key=True)
+    name = Column(String, unique=True)
+    type = Column(Enum(PackageType))
+    releases_status = relationship(
+        "CVERelease", secondary=package_release_status
```

Comment:
is it package_release_status or cve_release_status?

Reply:
So I created a different table called CVERelease, which is similar to USN release table, we can maybe later talk about this, but the data model didn't really fit into mine (I have a status and I have a "status description"), so I decided to create a new one to get it out of the way, so I can append data to the CVEs and let front-end carry on with work. We can revisit this.

Reply:
Sure, but the secondary table is still named `package_release_status` probably because this relationship used to be with the `Package` table.

Reply:
Is that not correct? And the `package_release_status` is related to the `cve_release` table. Things seem to be working fine here.